metaflow/cli.py

```diff
@@ -698,7 +699,7 @@ def run_on_kfp(obj,
 
     run_pipeline_result = create_run_on_kfp(obj.graph, code_url, experiment_name, run_name)
     echo("\nRun created successfully!\n")
-    echo("Run link: {0}{1}".format(RUN_LINK_PREFIX, run_pipeline_result.run_id))
+    echo("Run link: {0}{1}".format(KFP_RUN_URL_PREFIX, run_pipeline_result.run_id))
```

Comment:
tip: combining URLs would be done with `urljoin`, for example, one may forget to add a trailing "/"

```
>>> import urlparse
>>> urlparse.urljoin(url1, url2)
'http://127.0.0.1/test1/test4/test6.xml'
```
https://stackoverflow.com/questions/8223939/how-to-join-absolute-and-relative-urls

Reply:
Changed to using `posixpath.join` instead as urllib doesn't seem to be the right solution here. Tested both cases locally as well